packages/jaeger-ui/src/components/SearchTracePage/SearchResults/ScatterPlot.jsx

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
It would be good to undo this file's change and redo it such that Git would recognize the move, otherwise it's impossible to see what changed.